projects/angular/src/forms/datepicker/date-input.ts

```diff
@@ -192,6 +194,15 @@ export class ClrDateInput extends WrappedFormControl<ClrDateContainer> implement
     }
   }
 
+  private triggerControlValidation() {
+    if (this.datepickerHasFormControl()) {
+      this.control?.control?.updateValueAndValidity({ emitEvent: false }); // Added emitEvent purposefully inorder to prevent unnecessary valuechange triggers
+      if (this.control.control && this.control.control.touched) {
```

Comment:
```suggestion
      if (this.control.control?.touched) {
```

This can be simplified a bit.